migrations/v2.js

```diff
@@ -0,0 +1,143 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getComponents, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('adapt-contrib-accordion - v2.0.3 > v2.0.4', async () => {
+  let accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v2.0.3', { name: 'adapt-contrib-accordion', version: '>=2.0.0 <2.0.4' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = getComponents('accordion');
+    return accordions.length;
+  });
+
+  mutateContent('adapt-contrib-accordion - add accordion._items._classes attribute', async () => {
+    accordions.forEach(accordion => {
+      accordion._items.forEach(item => {
```

Comment:
A test with missing `_items` will error here.

```js
 { _id: 'c-110', _component: 'accordion' }
```
Is this ok?

Reply:
Adjusted based on 1:1 convo.